### ISSUE TITLE
Sixtyfour Convergence contrast for accessibilty

### ISSUE THREAD

**Issue by vv-monsalve** [jenskutilek/homecomputer-fonts#55]
@jenskutilek, @simoncozens, after a recent PR for another COLRv1 font, new requirements were introduced for our color fonts related to color contrast for accessibility standards.

This is the reference we were pointed to:

> [Our own guidelines on accessible text](https://fonts.google.com/knowledge/readability_and_accessibility/introducing_accessibility_in_typography) recommend:
> 
> "As the absolute bare minimum, the [W3C](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html) Web Content Accessibility Guidelines (WCAG) recommends the following contrast ratios for body and image text for an AA rating:
> - Large text (at 14 pt bold/18 pt regular and up) and graphics: 3:1 against the background
> - Small text: 4.5:1 against the background"
> 
> This is also reflected in guidelines on [material.io](http://material.io/) and our internal a11y guidelines. 

The above led to another requirement for the color fonts: dark and light mode color palettes are now mandatory.

<hr>

Sixtyfour Convergence font uses alphas that work perfectly for its purpose and in the dark mode, but at default values, they result in a pale pink-to-white appearance with a contrast issue in a light mode that seems like it doesn't meet the requirements.

So, I would suggest changing the default YELA value to around `35` to address this issue.

This small adjustment would not only help with the contrast concern but also communicate immediately what the font is about.

#### Current default

<img width="1367" alt="Screen Shot 2024-06-13 at 20 55 06" src="https://github.com/jenskutilek/homecomputer-fonts/assets/48698976/e91abde0-b810-4fbb-80da-aba748376743">

#### Proposed

<img width="1367" alt="Screen Shot 2024-06-13 at 20 56 08" src="https://github.com/jenskutilek/homecomputer-fonts/assets/48698976/0916c844-f709-4a7a-8f32-4d8f8d7748b0">
<img width="1367" alt="Screen Shot 2024-06-13 at 20 57 48" src="https://github.com/jenskutilek/homecomputer-fonts/assets/48698976/34196042-dc20-42d5-9c0d-ceb51ddcefbc">

<br>
<br>

 cc @davelab6

**Comment by simoncozens** [comment]
I think I might try to fix this another way, by supplying light and dark palettes, and making the light palette more contrasted.

**Comment by vv-monsalve** [comment]
Awesome! That would be even better. The pro or actual solution, indeed :)

**Comment by davelab6** [comment]
Excellent, thanks!

**Comment by vv-monsalve** [comment]
@simoncozens, after pulling the latest files in the repo at commit `6a6d378` and testing the font, the resulting color on the Light mode is still under the minimum contrast ratio 3:1 expected for a large text.

### Lightest color `FF9C92`

<img width="500" alt="Screen Shot 2024-06-19 at 12 07 13" src="https://github.com/jenskutilek/homecomputer-fonts/assets/48698976/809841b2-23dc-4f60-8085-c7acf7b20bea">

<br><br>

### Darkest color `FF7A70`

<img width="500" alt="Screen Shot 2024-06-19 at 12 07 23" src="https://github.com/jenskutilek/homecomputer-fonts/assets/48698976/1c683feb-b38c-4112-9419-78f6913972c1">

**Comment by vv-monsalve** [comment]
These are the colors I see after pulling the latest files
<img width="425" alt="Screen Shot 2024-06-20 at 16 02 39" src="https://github.com/jenskutilek/homecomputer-fonts/assets/48698976/c4e16727-77bb-4abc-a2e2-e7032763381d">

They look quite different from the ones in your image 
![image](https://github.com/jenskutilek/homecomputer-fonts/assets/48698976/4450e5f4-bd1b-4d05-9b06-55773efc4bf0)

**Comment by vv-monsalve** [comment]
It looks the same in FontGoogles :)
<img width="380" alt="Screen Shot 2024-06-21 at 9 42 24" src="https://github.com/jenskutilek/homecomputer-fonts/assets/48698976/1fa23797-200a-4c5c-9aef-9ecec9dc1585">

> It loads the default palette first regardless of background. 

AFAIK, this is the idea for the default palette. You can make it to match the Light one.

**Comment by simoncozens** [comment]
In this font, the default palette is for dark mode. Palette number 1 is for light mode. The application should choose the light mode palette if your background/OS is in light mode. FontGoggles doesn't change palette correctly, that's why I wrote sploot!

**Comment by vv-monsalve** [comment]
After our discussion during the meeting and a follow-up conversation with Dave, we agreed that given that the Light mode is still the default and most used one, we should make the fonts' default palette either match the Light palette or alternatively provide a third default option. However, the latter option is not applicable here, so making the light option the index 0 would be enough :)

**Comment by simoncozens** [comment]
I don't know why that failed. When you're looking at an error message you always need to look upwards from the end to find the nearest problem. In this case:

```
  File "/Users/viv_monsalve/Documents/01_GF/GF_Projects/homecomputer-fonts/venv/lib/python3.10/site-packages/paintcompiler/__init__.py", line 633, in compile_paints
    exec(python_code, this_locals, this_locals)
  File "<string>", line 10, in <module>
NameError: name 'SetColors' is not defined

[31/33] compress
fonttools ttLib.woff2 compress -o ../fonts/webfonts/Sixtyfour[BLED,SCAN].woff2 ../fonts/variable/Sixtyfour[BLED,SCAN].ttf
ninja: build stopped: subcommand failed.
make[1]: *** [build.stamp] Error 1
make: *** [build] Error 2
```

Why is `SetColors` not defined? I'm not sure but it was added to `paintcompiler` fairly recently. My guess is that you have an old `venv`, which has the older version of paintcompiler. So maybe delete your venv and try again?

**Comment by vv-monsalve** [comment]
> When you're looking at an error message you always need to look upwards from the end to find the nearest problem. In this case:

Yeah, I did that and saw the "SetColors" error, but also, above that line another failed jobs 
```
FAILED: /var/folders/j7/r59d5msn3_z7n6vmtdh32kzw0000gn/T/tmpy3gme7cq 
/Users/viv_monsalve/Documents/01_GF/GF_Projects/homecomputer-fonts/venv/bin/python3 -m gftools.builder.jobrunner paintcompiler --add-axis "XELA:-100:0:100:Horizontal Element Alignment" --add-axis "YELA:-100:0:100:Vertical Element Alignment" -o /var/folders/j7/r59d5msn3_z7n6vmtdh32kzw0000gn/T/tmpy3gme7cq /var/folders/j7/r59d5msn3_z7n6vmtdh32kzw0000gn/T/tmpct8x1dl4 
```

This confused me about the actual cause since I knew `SetColors` was added as part of your recent fixes to PaintCompiler. That's why I preferred to include the full traceback.

I don't run the `make build` command under a pre-created venv. However, I remembered now that I must delete it if I already have one (as you suggest here). I just re-ran the command and received the same `SetColors` error :-/

**Comment by simoncozens** [comment]
OK, I see the GitHub actions are failing too, so it's not a problem with your environment.

**Comment by simoncozens** [comment]
We need to bump t he paintcompiler version to 0.3.4 as well.

**Comment by vv-monsalve** [comment]
Okay, I updated it and successfully built the font, but the colors still contrast less than your above image in Light mode.

<img width="652" alt="Screen Shot 2024-06-26 at 11 14 21" src="https://github.com/jenskutilek/homecomputer-fonts/assets/48698976/f466d0c2-2867-452c-a2b9-13ef2088efd2">

**Comment by vv-monsalve** [comment]
Solved in #59

**Comment by jenskutilek** [comment]
Thank you!
